implementations/rust/ockam/ockam_command/src/project/create.rs

```diff
@@ -51,11 +54,10 @@ async fn run_impl(
     let space_id = space::config::try_get_space(&opts.config, &cmd.space_name)?;
     let node_name = start_embedded_node(ctx, &opts, None).await?;
     let mut rpc = RpcBuilder::new(ctx, &opts, &node_name).build();
-    rpc.request(api::project::create(
-        &cmd.project_name,
-        &space_id,
-        &cmd.cloud_opts.route(),
-    ))
+    rpc.request_with_timeout(
```

Comment:
I'm not sure if using longer timeouts on the command's RPC will do the effect we want. This will send a request to a NodeManager instance, which will in turn do a `ctx.send_and_receive` call with the default 30s timeout, receiving the timeout error on the command after 30s.

Maybe [retrying a few times like its done here](https://github.com/build-trust/ockam/blob/a1367541da9826f248a682baeed0b18936e63e87/implementations/rust/ockam/ockam_command/src/project/addon.rs#L452) would be more reliable.

Reply:
Just as an fyi, this PR updates the request in **both** places:

Command -> NodeManager, as well as NodeManager -> Orchestrator.

If this is a workable solution, both need to be updated

Reply:
> Command -> NodeManager, as well as NodeManager -> Orchestrator.

Yeah, that's what I didn't see, sorry. I thought the modified `Command -> NodeManager` endpoints were different from the `NodeManager -> Orchestrator` ones.